src/Tribe/Capabilities.php

```diff
@@ -1,38 +1,106 @@
 <?php
 
 class Tribe__Events__Capabilities {
+
+	/**
+	 * The transient key for delayed capabilities updates.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected $key_delayed_capabilities_update = '_tribe_events_delayed_capabilities_update';
+
+	/**
+	 * An array of roles to update capabilities.
+	 *
+	 * @since TBD
+	 *
+	 * @var array
+	 */
+	protected $roles = [
+		'administrator',
+		'editor',
+		'author',
+		'contributor',
+		'subscriber',
+	];
+
+	/**
+	 * An array of capabilities aliases by role.
+	 *
+	 * @since TBD
+	 *
+	 * @var array
+	 */
+	private $cap_aliases = [
+		'editor'      => [ // full permissions to a post type
+		                   'read',
+		                   'read_private_posts',
+		                   'edit_posts',
+		                   'edit_others_posts',
+		                   'edit_private_posts',
+		                   'edit_published_posts',
+		                   'delete_posts',
+		                   'delete_others_posts',
+		                   'delete_private_posts',
+		                   'delete_published_posts',
+		                   'publish_posts',
+		],
+		'author'      => [ // full permissions for content the user created
+		                   'read',
+		                   'edit_posts',
+		                   'edit_published_posts',
+		                   'delete_posts',
+		                   'delete_published_posts',
+		                   'publish_posts',
+		],
+		'contributor' => [ // create, but not publish
+		                   'read',
+		                   'edit_posts',
+		                   'delete_posts',
+		],
+		'subscriber'  => [ // read only
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 89b6dd01d70eee69f14d5c78317b0241</sub>